pages/public_cloud/containers_orchestration/managed_kubernetes/expose_your_applications_using_a_load_balancer/guide.en-gb.md

```diff
@@ -419,11 +427,104 @@ kubectl apply -f your-service-manifest.yaml
+metadata:
+  name: octavia-with-fixed-vip
+  annotations:
+    loadbalancer.ovhcloud.com/class: "octavia"
+    loadbalancer.openstack.org/port-id: "<openstack-port-uuid>"
+spec:
+  type: LoadBalancer
+```
+
+> [!warning]
+>
+> This feature is only supported when the MKS cluster is attached to a private network. (Public MKS is not supported)
+> The port needs to belong to the same network/subnet as the MKS cluster.
+> This feature is not compatible when the MKS cluster is configured with a LoadBalancerSubnetId (except if is the same as the NodeSubnetID).
+>
+
+### Restrict access for a LoadBalancer Service
+
+To apply IP restriction to the K8S LoadBalancer Service, you can define the array `.spec.loadBalancerSourceRanges` with a list of CIDR.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: octavia-ip-restrictions
+  annotations:
+    loadbalancer.ovhcloud.com/class: "octavia"
+spec:
+  loadBalancerSourceRanges:
+  - 1.2.3.4/32
+  - 5.6.7.0/24
+  type: LoadBalancer
+```
+
+If no value is assigned to this spec, no restriction will be applyed.
+
+### Sharing a Octavia LoadBalancer between multiple Kubernetes LoadBalancer Services
 
-#### Sharing load balancer with multiple Services
+You can share up to two Kubernetes Services on the same Octavia LoadBalancer. Those Services can be deployed on different MKS clusters (clusters must be in the same network).
+
+K8S services must expose different protocol/port (you cannot set the same protocol/port on both K8S Services). [Sharing load balancer with multiple Services (Github)](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md#sharing-load-balancer-with-multiple-services)
+
+To allow another K8S LoadBalancer Service to use an existing Octavia (created via MKS or via OpenStack), use the annotation `loadbalancer.openstack.org/load-balancer-id`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: octavia-basic-shared
+  annotations:
+    loadbalancer.ovhcloud.com/class: "octavia"
+    loadbalancer.openstack.org/load-balancer-id: "<existing-octavia-uuid>"
+spec:
+  type: LoadBalancer
+```
 
-By default, different Services of LoadBalancer type should have different corresponding cloud load balancers. However, the Cloud Controller Manager (CCM) allows multiple Services to share a single load balancer. We are currently working to provide this feature shortly. Official documentation: [Sharing load balancer with multiple Services](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md#sharing-load-balancer-with-multiple-services).
+> [!warning]
+>
+> If you want to delete a MKS cluster which is using SharedLoadBalancer feature, we strongly recommand you to delete K8S Service which using it to avoid any issue (like remaining Octavia LoadBalancer, configuration not removed, ...)
```

Comment:
```suggestion
> If you want to delete a MKS cluster which is using SharedLoadBalancer feature, we strongly recommend you to delete the K8S Service which using it to avoid any issue (like remaining Octavia LoadBalancer, configuration not removed, ...).
```